kong/router/atc.lua

```diff
@@ -554,6 +554,11 @@ function _M:exec(ctx)
 end
 
 
+function _M.get_schema()
+  return CACHED_SCHEMA
+end
```

Comment:
As it is a static table, won't be better to use the `_M.schema` format?

```suggestion
_M.schema = CACHED_SCHEMA
```